packages/react-examples/src/react-label/Label/Label.stories.tsx

```diff
@@ -55,60 +56,67 @@ export const CustomizableLabelExample = () => {
-    setStrong(checked!);
+  const updateStrong = (ev: React.FormEvent<HTMLInputElement>) => {
+    setStrong(!strong);
   };
 
-  const updateDisabled = (ev?: React.FormEvent<HTMLElement | HTMLInputElement>, checked?: boolean) => {
-    setDisabled(checked!);
+  const updateDisabled = (ev: React.FormEvent<HTMLInputElement>) => {
+    setDisabled(!disabled);
   };
 
-  const updateRequired = (ev?: React.FormEvent<HTMLElement | HTMLInputElement>, checked?: boolean) => {
-    setRequired(checked!);
+  const updateRequired = (ev: React.FormEvent<HTMLInputElement>) => {
+    setRequired(!required);
   };
 
-  const updateSize = (ev?: React.FormEvent<HTMLDivElement>, item?: IDropdownOption) => {
-    if (item && typeof item.key === 'string') {
-      setSize(item.key as LabelProps['size']);
-    }
+  const updateSize = (ev: React.FormEvent<HTMLSelectElement>) => {
+    setSize(ev.currentTarget.value as LabelProps['size']);
   };
 
   return (
     <div className={styles.exampleContainer}>
       <div className={styles.configurationContainer}>
-        <div>
+        <div className={styles.option}>
           <Label strong>Text</Label>
-          <TextField placeholder="I'm a label" onChange={updateLabelText} className={styles.textField} />
+          <input
+            type="text"
+            placeholder="I'm a label"
+            onChange={e => updateLabelText(e)}
+            className={styles.textField}
+          />
         </div>
         <div className={styles.checkbox}>
-          <Checkbox onChange={updateStrong} />
+          <input type="checkbox" onChange={e => updateStrong(e)} />
           <Label strong>Strong</Label>
         </div>
         <div className={styles.checkbox}>
-          <Checkbox onChange={updateDisabled} />
+          <input type="checkbox" onChange={e => updateDisabled(e)} />
           <Label strong>Disabled</Label>
         </div>
         <div className={styles.checkbox}>
-          <Checkbox onChange={updateRequired} />
+          <input type="checkbox" onChange={e => updateRequired(e)} />
           <Label strong>Required</Label>
         </div>
-        <div>
+        <div className={styles.option}>
           <Label strong>Required Text</Label>
-          <TextField onChange={updateRequiredText} className={styles.textField} />
+          <input type="text" onChange={e => updateRequiredText(e)} className={styles.textField} />
         </div>
-        <div>
+        <div className={styles.option}>
           <Label strong>Size</Label>
-          <Dropdown defaultSelectedKey={'medium'} options={SizeOptions} onChange={updateSize} />
+          <select defaultValue="medium" onChange={e => updateSize(e)}>
+            {sizeOptions.map(option => (
+              <option key={option} value={option}>
+                {option}
+              </option>
+            ))}
+          </select>
         </div>
       </div>
       <div className={styles.labelContainer}>
```

Comment:
@behowell 
> Actually a bunch of these state values don't appear to be used (`strong`, `disabled`, etc.)

They are being used in here

Reply:
It doesn't show up but, right after these lines there is a label with all the states.